e2e/rbd.go

```diff
@@ -4643,6 +4644,130 @@ var _ = Describe("RBD", func() {
 				validateOmapCount(f, 0, rbdType, defaultRBDPool, volumesType)
 			})
 
+			By("validate rbd image qos", func() {
+				qosParameters := map[string]string{
+					"BaseReadIops":            "2000",
+					"BaseWriteIops":           "1000",
+					"BaseReadBytesPerSecond":  "209715200",
+					"BaseWriteBytesPerSecond": "104857600",
```

Comment:
please define a var for numeric values as its used in two places to have single source

Reply:
updated